ZFSin/zfs/module/zfs/zfs_vnops_windows_lib.c

```diff
@@ -836,6 +916,128 @@ NTSTATUS CreateReparsePoint(POBJECT_ATTRIBUTES poa, LPCWSTR SubstituteName, LPCW
 	return status;
 }
 
+
+/*
+* go through all mointpoints (IOCTL_MOUNTMGR_QUERY_POINTS) and check if our driveletter is in the list
+* return 1 if yes, otherwise 0
+*/
+NTSTATUS mountmgr_is_driveletter_assigned(PDEVICE_OBJECT mountmgr, wchar_t driveletter, BOOLEAN *ret) {
```

Comment:
nit: we should probably stay with OpenZFS `cstyle.pl` coding standard, even if I have also been a bit lax with it :) "{" on new line for functions, and all that.